js/packages/eyes-cypress/src/browser/transformCypressConfig.ts

```diff
@@ -0,0 +1,56 @@
+import type {CypressEyesConfig} from '../expose'
+import type {SpecType, Config} from '@applitools/core'
+import {transformBrowsers, transformAccessibilityValidation, GetBatchProperies} from './utils'
+
+export function transformCypressConfig(config: CypressEyesConfig): Config<SpecType, 'ufg'> {
+  const getBatchProperty = GetBatchProperies(config)
+  return {
+    open: {
+      apiKey: config.apiKey,
+      serverUrl: config.serverUrl,
+      proxy: config.proxy,
+      appName: config.appName,
+      testName: config.testName,
+      displayName: config.displayName,
+      batch: {
+        id: getBatchProperty('batchId', 'id'),
+        name: getBatchProperty('batchName', 'name'),
+        sequenceName: getBatchProperty('batchSequenceName', 'sequenceName'),
+        notifyOnCompletion: getBatchProperty('notifyOnCompletion', 'notifyOnCompletion'),
+      },
```

Comment:
This way it will be much more clear. Also in your implementation, you are losing all other batch properties, for example, `properties`.
```suggestion
      batch: {
        ...config.batch,
        id: config.batchId ?? config.batch?.id,
        name: config.batchName ?? config.batch?.name,
        sequenceName: config.batchSequenceName ?? config.batch?.sequenceName,
        notifyOnCompletion: conifg.notifyOnCompletion ?? config.batch?.notifyOnCompletion,
      },
```